pkg/gitproviders/common.go

```diff
@@ -71,14 +94,64 @@ func getOwnerType(provider gitprovider.Client, owner string) (string, error) {
 	})
 
 	if err != nil {
-		if errors.Is(err, gitprovider.ErrNotFound) {
-			return OwnerTypeUser, nil
+		if errors.Is(err, gitprovider.ErrNotFound) ||
+			strings.Contains(err.Error(), badCredentialsError) ||
+			strings.Contains(err.Error(), notFoundError) {
```

Comment:
you are checking again the not found error, and bad credentials is an actual error, so it should return as such.

Reply:
If you are trying to get a org using a user account that doesn't exist it responds with bad credentials. 
<img width="838" alt="image" src="https://user-images.githubusercontent.com/3892786/120665413-5c9b2a80-c451-11eb-9d7c-96ae79296b85.png">

For the Not found issue, I guess there is a bug on gogitprovider because it is not caching the error with this condition `errors.Is(err, gitprovider.ErrNotFound)`

Reply:
I see, so we might need to find a different way, or another endpoint, imagine if the person actually uses the wrong token to query the org, that would return the wrong type, creating another sort of bugs.

Reply:
btw, I've just tested and returned as expected the Not Found.

![image](https://user-images.githubusercontent.com/355751/120679050-1f459580-c46f-11eb-9dd1-863a0938bd93.png)

Reply:
did you use your personal token for this test?

Reply:
I used the same org as you here:
<img width="839" alt="image" src="https://user-images.githubusercontent.com/3892786/120684467-5498b600-c464-11eb-9d85-62ae4f39d963.png">